Libplanet.Tests/Action/Sys/SetValidatorTest.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using System.Numerics;
+using Bencodex.Types;
+using Libplanet.Action;
+using Libplanet.Action.Sys;
+using Libplanet.Assets;
+using Libplanet.Blocks;
+using Libplanet.Consensus;
+using Libplanet.Crypto;
+using Xunit;
+using static Libplanet.Tests.TestUtils;
+using Random = System.Random;
+
+namespace Libplanet.Tests.Action.Sys
+{
+    public class SetValidatorTest
+    {
+        [Fact]
+        public void Constructor()
+        {
+            var publicKey = new PrivateKey().PublicKey;
+            var power = new BigInteger(1);
+            var validator = new Validator(publicKey, power);
+            var action = new SetValidator(validator);
+            Assert.Equal(publicKey, action.ValidatorPublicKey);
+            Assert.Equal(power, action.Power);
+        }
+
+        [Fact]
+        public void Serialize()
+        {
+            var publicKey = new PrivateKey().PublicKey;
+            var power = new BigInteger(1);
+            var validator = new Validator(publicKey, power);
+            var action = new SetValidator(validator);
+            AssertBencodexEqual(
+                Bencodex.Types.Dictionary.Empty
+                    .Add("validator_public_key", publicKey.Format(false))
+                    .Add("power", new BigInteger(1).ToByteArray()),
+                action.PlainValue
+            );
+        }
+
+        [Fact]
+        public void Deserialize()
+        {
+            var publicKey = new PrivateKey().PublicKey;
+            var power = new BigInteger(1);
+            IValue serialized = Bencodex.Types.Dictionary.Empty
+                .Add("validator_public_key", publicKey.Format(false))
+                .Add("power", new BigInteger(1).ToByteArray());
+            var action = new SetValidator();
+            action.LoadPlainValue(serialized);
+            Assert.Equal(publicKey, action.ValidatorPublicKey);
+            Assert.Equal(power, action.Power);
+        }
+
+        [Fact]
+        public void Execute()
+        {
+            var random = new Random();
+            Address signer = random.NextAddress();
+            Currency bazCurrency = Currency.Uncapped("BAZ", 0, random.NextAddress());
+            var prevStates = new AccountStateDeltaImpl(
+                accountStateGetter: addr => new IValue[addr.Count],
+                accountBalanceGetter: (addr, c) => c * 0,
+                totalSupplyGetter: c => c * 0,
+                validatorSetGetter: () => new ValidatorSet(),
+                signer: signer
+            );
```

Comment:
May be some `AccountStateDeltaImpl` mockup fixture could be needed?
This `prevStates` construction seems to be repeated on `Libplanet.Tests.Action.Sys`.
Defining parent test base class for system actions seems to be better for me.